test/e2e/framework/framework.go

```diff
@@ -206,16 +117,68 @@ func KubernetesClientEnv(t *testing.T) []string {
 	return []string{"KUBERNETES_SERVICE_HOST=" + host, "KUBERNETES_SERVICE_PORT=" + port}
 }
 
+type Accessory interface {
+	Ready()
+	Bind(cmd CiOperatorCommand)
+}
+
+var _ Accessory = &accessory{}
+
+type accessory struct {
+	command    string
+	port       string
+	healthPort string
+
+	binder func(port string, cmd CiOperatorCommand)
```

Comment:
This would be easier to reason about if the `accessory` just had a `CIOperatorFlags` method rather than getting a `CiOperatorCommand` and then manipulating that. The CIOperatorCommand depends on the accessory, not the other way round like the current implementation suggests